src/Security/MemberAuthenticator/CookieAuthenticationHandler.php

```diff
@@ -189,13 +216,14 @@ public function logIn(Member $member, $persistent = false, HTTPRequest $request
             $rememberLoginHash = RememberLoginHash::generate($member);
             $tokenExpiryDays = RememberLoginHash::config()->uninherited('token_expiry_days');
             $deviceExpiryDays = RememberLoginHash::config()->uninherited('device_expiry_days');
+            $secure = $this->getTokenCookieSecure();
```

Comment:
Should this use the current request as well? like we do for the session cookie. "if the request is HTTPS && we use the secure cookie setting"?

Reply:
At the moment this assumes that when you use this setting, your site unconditionally redirects to HTTPS for all routes. We could make it more friendly towards mixed mode (login form submitting to HTTP rather than HTTPS), but there's all kinds of other edge cases that we've discussed in the original ticket. I'd prefer if we document this as "only turn on option if your whole site is on HTTPS".

Reply:
OK, that makes sense